Download seq branch

vtorda opened the pull request:
Updated sequence downloading script.

vtorda committed:
TidyUp Script_1SingleRun

Writing out plain text characters acquired by using the entrez_fetch function was simplified: instead of performing a complicated conversion the write function that is using cat can do the job easily

vtorda committed:
StartedToWorkOnDowloadScripts

I deleted other scripts from this branch since I don't want to modify them.

vtorda committed:
Update RSetup.R

vtorda committed:
Merge branch 'main' into Script_1_TordaEditing

vtorda committed:
Rewrote the sequence download script

I rewrote the 1_Download_seqs.R script: I am using web_history object all in all steps. sequences were placed in a file more easily than in the original script.

vtorda committed:
Kept only the download seq script

Rowena-h commented on 1_Download_seqs.R:
```diff
@@ -5,86 +5,76 @@
+      if(j < batch_n){
+        cat("\n",seq_start[j]+49, "sequences downloaded\r")
+      }else{
+        cat("\n", max_seq, "sequences downloaded\r")
```
I found that the API warning message cuts off this final message printing the total number of sequences downloaded - I think this can simply be fixed by changing `sequences downloaded\r` to `sequences downloaded\n`

Rowena-h commented on 1_Download_seqs.R:
```diff
@@ -5,86 +5,76 @@
       }
+    }
+    # write out metadata
+    write.table(x = meta_df, file = paste0(path_to_output_dir, search_term, ".metadata"),
```
Just a suggestion to add `.tsv`  after `.metadata` so the file is recognised by editors

vtorda committed:
Update 1_Download_seqs.R

Changed end line characters and add .tsv to the name of the tsv as Rowena suggested

vtorda commented:
I am going to delete this branch
